client/components/app-promo/index.jsx

```diff
@@ -69,15 +75,26 @@ export default React.createClass( {
 			return null;
 		}
 
-		var promo_link = 'https://apps.wordpress.com/desktop/?ref=promo_' + this.props.location + '_' + this.state.promo_item.promo_code;
-		var element = (
+		const promo_link = 'https://apps.wordpress.com/desktop/?ref=promo_' + this.props.location + '_' + this.state.promo_item.promo_code;
+		const element = (
 			<div className="app-promo">
```

Comment:
How should I approach the warning `className should follow CSS namespace guidelines (expected app-promo__ prefix)` that gets triggered here?
As explained in the [eslint-plugin-wpcalypso docs](https://github.com/Automattic/eslint-plugin-wpcalypso/blob/master/docs/rules/jsx-classname-namespace.md#rule-details), this should be correct.